hooks/post/30-upgrade.rb

```diff
@@ -1,3 +1,7 @@
+require 'fileutils'
+
+STEP_DIRECTORY='/etc/foreman-installer/hooks/post/'
```

Comment:
Can we use a directory out of `@kafo.config.app`? Just in case we stop using this dir.

Or, what do you think about having a hidden directory/file like we do for applied migrations? Maybe right inside the hook directory itself, we could have a .ran file or something.